python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -36,7 +36,7 @@
 from scipy.stats import binned_statistic_2d, binned_statistic_dd
 
 from ...interfaces import KeyedData, KeyedDataSchema, PlotAction, Scalar, Vector
-from ...statistics import nansigmaMad
+from ...statistics import nanmedian, nansigmaMad
```

Comment:
I think nanMedian and nanSigmaMad would be easier to read

Reply:
I agree, but I was following the numpy naming convention. Having said that, it might make more sense to not follow numpy so as to distinguish them instead?